.github/workflows/cert.yml

```
@@ -33,21 +33,28 @@ jobs:
       if: steps.check_changes.outcome == 'success'
       with:
         ocaml-compiler: ${{ env.ocaml-compiler }}
-    - run: |
-        opam install castore x509 ca-certs cstruct mdx ocamlformat
+
+    - name: Install OCaml deps
+      if: steps.check_changes.outcome == 'success'
+      run: |
+        opam install castore x509 ca-certs cstruct mdx ocamlformat dune-release
         opam exec -- dune build
         ./_build/default/regen.exe
         opam exec -- dune fmt || true
 
     - name: Bump version and create release
       if: steps.check_changes.outcome == 'success'
+      shell: bash
       run: |
-        current_version=$(git describe --tags --abbrev=0)
+        git pull --tags --force
+        current_version=$(git describe --tags $(git rev-list --tags --max-count=1))
         IFS='.' read -r major minor patch <<< "$current_version"
         new_version="$major.$minor.$((patch + 1))"
-        
-        git tag "$new_version"
-        git push origin "$new_version"
-        gh release create "$new_version" --generate-notes
+
+        opam exec -- dune-release tag "$new_version"
+        opam exec -- dune-release distrib
+        opam exec -- dune-release publish distrib -y
+        opam exec -- dune-release opam pkg
+        opam exec -- dune-release opam submit
```

I think we may be able to use `dune-release` on its own, like:

```suggestion
        git tag "$new_version"
        yes | opam exec -- dune-release --no-auto-open
```

Since `dune-release` behaves like: distrib + publish + opam pkg + opam submit

I may be wrong tho, since I haven't tried using dune-release in a non-interactive way.